WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -284,5 +285,49 @@ public static async Task<ISet<uint256>> GetAllDependentsAsync(this RPCClient rpc
 
 			return hashSet;
 		}
+
+		public static async Task<VerboseBlockInfo> GetVerboseBlockAsync(this IRPCClient rpc, uint256 blockId)
+		{
+			var resp = await rpc.SendCommandAsync(RPCOperations.getblock, blockId, 3, false).ConfigureAwait(false);
```

Comment:
There is no such param.

```suggestion
			var resp = await rpc.SendCommandAsync(RPCOperations.getblock, blockId, 3).ConfigureAwait(false);
```